requirements/testing.txt

```diff
@@ -2,7 +2,7 @@
 -r local-docs-build.txt
 
 django-dynamic-fixture==2.0.0
-pytest==4.2.0
+pytest==4.2.1
```

Comment:
We need to merge this  https://github.com/rtfd/readthedocs.org/pull/5294 before

Reply:
We can fix the pytest issue on that PR instead of blocking this one.